configure.ac

```diff
@@ -1,6 +1,8 @@
 # ======================== initialization ===============================
 
-AC_INIT([SIPp], [3.5.0-rc1], [sipp-users@lists.sourceforge.net], [sipp])
+AC_INIT([SIPp],
+        [m4_esyscmd_s([git describe --tags --always --dirty])],
```

Comment:
Yeah, but this isn't for that. This is meant for helping releases/debugging people who build from git:

1. We release a tarball. To do that sanely, we should run the first two steps of `build.sh`: `git submodule update --init` and `autoreconf -vifs` and tar up the result (without `.git` included obviously). The resulting `./configure` script is now setup with a version pulled from git describe. No manual tweaking of code before tagging a release.
2. If someone builds straight from git (like if I was to package a -git version of sipp internally), the `./sipp -v` will be reflective of HEAD when it was built.

I don't really see this as useful for manual in-tree work since the usefulness of this goes away if the tree is dirty anyways.